gui/view/bigtable.cpp

```diff
@@ -26,8 +26,8 @@
 //!
 //! The first column contains row numbers. The remaining numCols columns contain data.
 
-BigtableModel::BigtableModel()
-    : TableModel("data#")
```

Comment:
Das "#" brauchen wir, um verschiedene Instanzen (mainframe und popup) zu unterscheiden.

Reply:
die BigtableModel Instanzen im Popup Dialog bekommen eine fortlaufende Nummer an ihren Namen angehängt:
``` c++
    static int datatableCounter = -1;
    datatableCounter++;
    auto* bigtableView = new BigtableView("datatable" + QString::number(datatableCounter));
```
in [gui/dialogs/popup_bigtable.cpp](https://github.com/scgmlz/Steca/pull/176/files#diff-a9208e917f975c4da77a719fed4fa971)

Reply:
Damit hätten wir zwei Hochzählmechanismen: hier und in Qcr. Geht das nicht aus einem Guss?

Reply:
`TableView` ist das Qcr Object, aber `TableModel` logt die Events.
``` c++
gConsole->log(name() + " highlight " + QString::number(row));
```
das führt beim abspielen eines Logfiles zu einem `unknown command 'data#'` Fehler